### PR TITLE
cleaned up uiowa_acquia_authentication_form_alter

### PR DESCRIPTION
**Pull request by adamdelaney**
Unset un-needed form items and removed css

### CODE REVIEW

**Comment by adamdelaney** [comment]
Instead of disabling the form inputs that we don't need and hide them with css, we should probably just unset them.

**Comment by brandonneil** [comment]
It might be better to use the #access property of Form API instead of unset in case the form is also altered by other modules it doesn't loose those form elements: https://api.drupal.org/api/drupal/developer!topics!forms_api_reference.html/7.x#access

File: uiowa_acquia_authentication.module [hunk 1]
```diff
@@ -54,12 +54,10 @@ function uiowa_acquia_authentication_form_alter(&$form, &$form_state, $form_id)
         '#weight' => -1,
       );
 
-      $form['name']['#disabled'] = TRUE;
-      $form['pass']['#disabled'] = TRUE;
-
-      $form['actions']['submit']['#disabled'] = TRUE;
-
-      $form['#attached']['css'][] = drupal_get_path('module', 'uiowa_acquia_authentication') . '/uiowa_acquia_authentication.css';
+      // Unset name, password and action inputs.
```
Comment (adamdelaney) [inline uiowa_acquia_authentication.module]: @briand44 Can you take a look at this. Is there a reason we need this input fields?

**Comment by briand44** [comment]
I believe Rick and I discussed this at the time and decided to go with disabling them and hiding with css but I don't recall why. At the moment I can't think of any issues with setting the access to false, so I think this is fine.

**Pull request merged**